src/sbvr-api/uri-parser.coffee

```diff
@@ -91,13 +97,9 @@ exports.parseOData = (b) ->
 			{ url, apiRoot } = splitApiRoot(b.url)
 			odata = memoizedParseOdata(url)
 
-			# if we parse a canAccess action rewrite the resource to ensure we
-			# do not run the resource hooks
-			if odata.tree.property?.resource == 'canAccess'
-				odata.tree.resource = odata.tree.resource + '#' + odata.tree.property.resource
-
 			return {
 				method: b.method
+				url
```

Comment:
NIT: would rather have `url: url` for consistency

Reply:
It would be more consistent with the other entries in this specific object because they all use a different key, but it's consistent with what we usually do in other places with omitting the `: x` if we have `x: x` (especially in typescript where we always have the `{}`)